java/runtime/src/main/java/io/ray/runtime/AbstractRayRuntime.java

```diff
@@ -178,6 +179,15 @@ public ObjectRef call(PyFunction pyFunction, Object[] args, CallOptions options)
         functionDescriptor, args, /*returnType=*/ Optional.of(pyFunction.returnType), options);
   }
 
+  @Override
+  public ObjectRef call(CppFunction cppFunction, Object[] args, CallOptions options) {
+    CppFunctionDescriptor functionDescriptor =
+        new CppFunctionDescriptor(cppFunction.functionName, "JAVA", "");
+    // Python functions always have a return value, even if it's `None`.
```

Comment:
This comment is copy-pasted. You should change it according to the C++ case. 
But I think the Python one is already out-dated. You can remove both of them.

Reply:
ok